src/js/actions/type.js

```diff
@@ -202,7 +202,7 @@ define(function (require, exports) {
      * @param {boolean=} coalesce Whether to coalesce this operation's history state
      * @return {Promise}
      */
-    var updateColor = function (document, layers, color, coalesce) {
+    var updateColor = function (document, layers, color, coalesce, fromSet) {
```

Comment:
nit: this param name is not very meaningful.  Maybe `amendHistory` ?

also: needs to be added to JSDoc